bindings/dart/lib/src/common/reranker/ai.dart

```diff
@@ -76,15 +77,14 @@ class XaynAi {
   void free() => throw UnsupportedError('Unsupported platform.');
 }
 
+/// Maximum number of threads to be used for parallel features.
+const int maxNumberOfThreads = 16;
+
 /// Selects the number of threads used by the [`XaynAi`] thread pool.
 ///
 /// On a single core system the thread pool consists of only one thread.
 /// On a multicore system the thread pool consists of
-/// (the number of logical cores - 1) threads.
-int selectThreadPoolSize(int numberOfProcessors) {
-  if (numberOfProcessors > 1) {
-    return numberOfProcessors - 1;
-  } else {
-    return numberOfProcessors;
-  }
-}
+/// (the number of logical cores - 1) threads, but at most [`maxNumberOfThreads`] threads.
```

Comment:
and at least one.

Reply:
the sentence refers to multi-core systems, hence i'd say the `(the number of logical cores - 1) threads` already implies at least one thread. i can still add it for explicitness.

Reply:
ok, with the diff I missed that was connected with the previous comment.